### PR TITLE
docs: DS-11789: Adds the common questions

### PR DESCRIPTION
**Pull request by bredamc**
## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by jbyrne-redhat — APPROVED** [review]
Looks good, Breda. 

I do suggest one change, but am approving nonetheless.

File: common-questions.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:_module-type: REFERENCE
+
+[id="common-questions_{context}"]
+= Common questions
+
+[role="_abstract"]
+In addition to documentation, Red Hat provides several "how-to" documents that answer common questions a data scientist might have as they work.
+
+The *Resources* page of the {productname-short} dashboard has a rich set of learning resources for {productname-short} and supported applications. You can use the category links on the page to filter the resources for various stages of your data science workflow.
```
[the remark this answers is not in the document]
Comment (bredamc) [reply common-questions.adoc:9]: I respectfully disagree :) Thank you for approving regardless :D

[the remark this answers is not in the document]
Comment (jbyrne-redhat) [reply common-questions.adoc:9]: Hmmm. They just seemed like two disconnected statements, saying two different things.

There are other information types  on the **Resources** page besides how-tos, so the first paragraph seems unnecessarily limiting. 

For that reason, it seemed better to integrate the two.

[the remark this answers is not in the document]
Comment (bredamc) [reply common-questions.adoc:9]: Updates made in PR 27 -- thanks, J!

**Pull request merged**